src/apps/activity_assignments/crud/assignments.py

```diff
@@ -71,9 +185,18 @@ async def get_by_applet(self, applet_id: uuid.UUID, query_params: QueryParams) -
         query = (
             select(ActivityAssigmentSchema)
             .outerjoin(ActivitySchema, ActivitySchema.id == ActivityAssigmentSchema.activity_id)
-            .outerjoin(ActivityFlowSchema, ActivityFlowSchema.id == ActivityAssigmentSchema.activity_flow_id)
-            .join(respondent_schema, respondent_schema.id == ActivityAssigmentSchema.respondent_subject_id)
-            .join(target_schema, target_schema.id == ActivityAssigmentSchema.target_subject_id)
+            .outerjoin(
+                ActivityFlowSchema,
+                ActivityFlowSchema.id == ActivityAssigmentSchema.activity_flow_id,
+            )
```

Comment:
why you add this comma at end here? maybe you should review your local lint configuration

Reply:
trailing comma is needed when you break the line and if you add a trailling comma it wiil force lint to break the line.  In this case a line break is optional, so trailling comma is also optional

Reply:
I'm still not following, I'm afraid, sorry! Could you share with me an example in the codebase where the trailing comma on the last line is not present?

Also, why do we still have code style issues that `ruff` is not catching?

Reply:
For posterity: @rcmerlo shared [a Loom](https://www.loom.com/share/e309dd679d7d4e0496ae3898a5d1348c?sid=5bf3dcfe-3f45-4cc4-928e-998ba950019e) explaining his concern. To summarize:

The trailing comma here isn't needed, and including it is causing the formatter to place each parameter on a separate line. If the trailing comma is deleted, the formatter should keep the parameters on the same line, because it would fit under the line length, which is preferred whenever possible.

Reply:
Yup, there are many places in `*.py` files where exactly 120 characters are present (you can do a project-wide search for the regex `.{120}` to find examples). Sometimes lines exceed that number if it's not possible to word-wrap (for example, long strings). My VS Code formatter is already word-wrapping at 120 characters when possible; I think it's always just been using the `pyproject.toml` file. So you should be good.

Reply:
I installed the Ruff extension and apply ruff check . && ruff format --check . && isort --check .  to detect unformated files and ruff format to change files.